src/js/tracks/audio-track-list.js

```diff
@@ -36,7 +36,7 @@ class AudioTrackList extends TrackList {
   /**
    * Create an instance of this class.
    *
-   * @param {AudioTrack[]} [tracks=[]]
+   * @param { import('./audio-track').default[] } [tracks=[]]
```

Comment:
I think I'm okay with these kinds of changes as a temporary fix to make TypeScript work, but I have to admit that I'm concerned that it makes the documentation comments and (in some cases) may make the rendered API docs less useful.

We should find a better way to implement TypeScript compatibility for the long-term.

Reply:
The best way would be to rewrite `video,js` in TypeScript. :smile: